Form/Type/CollectionType.php

```diff
@@ -24,14 +23,15 @@ class CollectionType extends AbstractType
 {
     /**
      * {@inheritdoc}
+     *
+     * @todo BC code for Symfony < 2.8 . To be removed.
      */
     public function getParent()
     {
-        if (version_compare(Kernel::VERSION, '2.8.0', '>=')) {
-            return 'Symfony\Component\Form\Extension\Core\Type\CollectionType';
-        } else {
-            return 'collection';
-        }
+        return
+            method_exists('Symfony\Component\Form\FormTypeInterface', 'setDefaultOptions') ?
```

Comment:
@stof are you referring to [this file](https://github.com/symfony/symfony/blob/master/UPGRADE-3.0.md)? I can't find the recommendation you talk about in it (although it does talk about `getBlockPrefix`)